src/main/rx/spektrum.h

```diff
@@ -20,5 +20,5 @@
 #define SPEKTRUM_SAT_BIND_DISABLED 0
 #define SPEKTRUM_SAT_BIND_MAX 10
 
-uint8_t spektrumFrameStatus(void);
```

Comment:
I agree, its going to be a nightmare to have to keep straight which of my quads I have updated or not.  There should just be a wizard in the configurator to figure out what the endpoints are set to, rather then making a chance thats going to break something everyone already is using one way.